README.rst

```diff
@@ -78,6 +78,12 @@ Analyzing the Images
 -  **N.B.:** You should place your plates such that A1 is in the upper
    right corner. Otherwise you should add the argument
    ``--orientation bottom-left`` to the above commands.
+-  You can choose to analyse individual plates only by specifying the
+  ``--trays`` parameter one or more times:
```

Comment:
Would it maybe be simpler to allow something like `--trays 1-6,9` instead of needing mulitple trays flags?

Reply:
`1-6,9` is a bad example since that would of course be invalid, but the e.g. `--trays 1-3,5`

Reply:
Or maybe even just `--trays 1,2,3,5`

Reply:
Yeah, I was thinking about that. How about a full listing as a compromise (`1,2,3,4,5,6`), though? Since I don't really feel like parsing the various combinations of ranges and individual numbers.

Reply:
Yes a full list would be perfectly fine for me